docker/luna_tutorial/README.md

```diff
@@ -0,0 +1,13 @@
+Luna Tutorial
+=============
+See make help for instructions on how to build and run the tutorial docker container. 
+
+```
+$ make help
+```
+
+In order to run this tutorial, first build the docker image and then lauch the container. Once the container is lauched, copy the url in the container's terminal log and paste it in a browser to open the jupyterlab environment containing the tutorial notebooks. Then step through the notebooks. Note that the url by default is set to localhost. If you need to run the tutorial from a remote system, replace the localhost IP address with the remote system IP address in the Makefile run target and rerun the container.  
```

Comment:
we could have a HOST or IP as an arg to make this edit easier.

Reply:
Good call! Added with localhost as default.